common-app-lib/project.clj

```diff
@@ -26,7 +22,7 @@
   :profiles {:security {:plugins [[com.livingsocial/lein-dependency-check "1.4.1"]]
                         :dependency-check {:output-format [:all]
                                            :suppression-file "resources/security/suppression.xml"}}
-             :dev {:exclusions [[org.clojure/tools.nrepl]]
+             :dev {:exclusions []
```

Comment:
I'm guessing that this is okay because the tools.nrepl was only loaded in the dev profile, so there really is no reason to exclude it as the code never gets into the jar?

Reply:
I was having issues even running CMR locally because nrepl packages weren't available while it was listed in the exclusion list